packages/experiments/src/components/TilesList/TilesList.tsx

```diff
@@ -122,11 +125,37 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
           role={role ? 'presentation' : undefined}
           getPageSpecification={this._getPageSpecification}
           onRenderPage={this._onRenderPage}
+          ref={this.listRef}
+          usePageCache={true}
+          {...this.props.listProps}
         />
       </FocusZone>
     );
   }
 
+  public scrollToIndex(index: number, mode: ScrollToMode = ScrollToMode.auto): void {
+    if (this.listRef && this.listRef.current) {
```

Comment:
Nit: you can probably refactor out the `this.listRef` portion of the conditional when assigning the member field

Reply:
removed the this.listRef in the conditional